step-templates/json-merge.json

```diff
@@ -0,0 +1,69 @@
+    "Octopus.Action.Script.ScriptFileName": null,
+    "Octopus.Action.Package.FeedId": null,
+    "Octopus.Action.Package.PackageId": null
+  },
+  "Parameters": [
+    {
+      "Id": "ff6ebb11-b388-460b-835e-5cf98ae394ed",
+      "Name": "SourceFile",
```

Comment:
Could you please prefix the parameter names with `jmf_` (and update the usages in the script body) to minimize the risk of duplicate variables in a project.